config/application.rb

```diff
@@ -120,5 +120,10 @@ def preview?
     def debug?
       Rails.env.development? || review?
     end
+
+    # @return [ActiveSupport::TimeWithZone] public beta launch date and time
```

Comment:
The method name is self-documenting now so the comment can be used to provide something the code does not.